week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -7,7 +7,69 @@ In this homework, you are going to work with an ecommerce database. In this data
 Below you will find a set of tasks for you to complete to set up a database for an e-commerce app.
 
 To submit this homework write the correct commands for each question here:
+
 ```sql
+1-select name,address
+   from customers c
+where country ='United States';
+
+2-select *
+  from customers c
+   order by name;
+
+3-select *
+from products p
+where  lower(product_name) like '%socks%';
+
+4-select p.id ,p.product_name ,pa.unit_price ,pa.supp_id
+from products p  join product_availability pa on p.id =pa.prod_id ;
```

Comment:
you also need to check that the unit price is > 100 - you can use a where clause to do this